easybuild/easyconfigs/b/Bader/Bader-1.0.3-intel-2018a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'MakeCp'
+
+name = 'Bader'
+version = '1.0.3'
+
+homepage = 'http://theory.cm.utexas.edu/henkelman/code/bader/'
+description = "A fast algorithm for doing Bader's analysis on a charge density grid."
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['http://theory.cm.utexas.edu/henkelman/code/bader/download/']
+sources = ['bader.tar.gz']
+checksums = ['9defc5d005521a3ed8bad7a3e2e557283962269eddf9994864c77b328c179c63']
```

Comment:
@migueldiascosta Checksum failed for me, I guess that shouldn't surprise me since the tarball is unversioned...

Can we contact the author and request versioned tarballs?

Reply:
@boegel yeah, I only noticed after submitting (also, this was a test case for my auto-labels pr before I remembered to use `pr-target-account` :p, but might as well finish it)

the archived versions are versioned (well, in versioned subfolders), the problem is that "only" with the most recent one. I'll change this PR to version 1.0.2 and then try to contact the author.

Reply:
I had forgotten to contact the author, tried now, let's wait for a reply

Reply:
Any news?

Reply:
@boegel yes, there is now a versioned url to the latest version (although it seems that in the process they repackaged it, checksum changed, see #7804...)